app/domain/services/create_course/service.rb

```diff
@@ -0,0 +1,45 @@
+class Services::CreateCourse::Service
+  def process(course_uuid:, ecosystem_uuid:)
+
+    start_time     = Time.now
+    start_time_str = start_time.utc.iso8601(6)
+
+    ecosystem = Ecosystem.find_by uuid: ecosystem_uuid
```

Comment:
Very minor, but IMO this could be inlined below, since it's not used anywhere else:
```ruby
  unless Ecosystem.find_by uuid: ecosystem_uuid
```